src/dispersion.h

```diff
@@ -5,6 +5,9 @@
 #include <iosfwd>
 #include <vector>
 
+// Note: 4/30/2019, changing dispersion sources to reflect a normal 
```

Comment:
This comment is unnecessary, the commit and PR histories are a more complete view into this kind of historical change over time, and in-source notes about it just go out of date.  If you want this specific change documented, put it in a commit message.